### PR TITLE
New Metals, Alloys and Mineral Types, Part 1!

### PR DESCRIPTION
**Pull request by Koollan**
:cl:
rscadd: I added some metals and mineral types in order to actually acquire the new metals. This is the first pass. Check the PR for full details.
/:cl:

**Metals added:**
Zinc
Tin
Lead

**Alloys added:**
Brass (Copper + Zinc)
Bronze (Copper + Tin)

**Minerals added:** (Left to right is richness of metal)
Magnetite (Iron, Copper, Sulfur)
Chalcopyrite (Copper, Sulfur, Iron)
Dense Graphite (Carbon, Plasticide, Acetone, Diamond) _// Was previously Oil Shale_
Cassiterite (Tin, Tungsten)
Wolframite (Tungsten, Iron, Tin)
Sperrylite (Platinum, Iron, Glass, Titanium)
Sphalerite (Zinc, Iron, Silicon)
Galena (Lead, Iron, Silver)
Calaverite (Gold, Silver)

### CODE REVIEW

**Comment by MistakeNot4892** [comment]
Only issue I have with the materials list is oil shale as kerogen needs plants etc. to form (which is why coal got renamed).

**Comment by MistakeNot4892** [comment]
Only issue I have with the materials list is oil shale as kerogen needs plants etc. to form (which is why coal got renamed).

File: code/modules/materials/definitions/solids/materials_solid_metal.dm [hunk 1]
```diff
@@ -53,12 +53,27 @@
...
+	stack_type = /obj/item/stack/material/brass
+	color = "#dab900"
+	reflectiveness = MAT_VALUE_VERY_SHINY
+	sheet_singular_name = "sheet"
+	sheet_plural_name = "sheets"
 	value = 1.2
 
 /decl/material/solid/metal/copper
 	name = "copper"
+	lore_text = "A metal used in some components and many alloys. Known for its color shifting properties when oxidized."
```
Comment (MistakeNot4892) [inline code/modules/materials/definitions/solids/materials_solid_metal.dm]: Probably `color-shifting`.

File: code/modules/materials/definitions/solids/materials_solid_mineral.dm [hunk 2]
```diff
@@ -290,4 +290,236 @@
...
+	ore_icon_overlay = "lump"
+	ore_type_value = ORE_SURFACE
+	ore_data_value = 1
+	value = 0.8
+	sparse_material_weight = 25
+	rich_material_weight = 15
+	fuel_value = 0.8
+	dissolves_into = list(
+		/decl/material/solid/metal/zinc = 0.7,
+		/decl/material/solid/metal/iron = 0.2
```
Comment (MistakeNot4892) [inline code/modules/materials/definitions/solids/materials_solid_mineral.dm]: Needs a comma.

Comment (Koollan) [reply code/modules/materials/definitions/solids/materials_solid_mineral.dm]: Resolving all of the above shortly. I called oil shale 'Dense Graphite' before, so I'm gonna rename it to that

**Review by MistakeNot4892 — APPROVED** [review]

**Pull request merged**